src/applications/vaos/tests/new-appointment/components/VAFacilityPage/VAFacilityPageV2.multi.unit.spec.jsx

```diff
@@ -62,103 +64,75 @@ const parentSite984 = {
-    },
-  },
-  {
-    id: '983GE',
-    attributes: {
-      ...getFacilityMock().attributes,
-      institutionCode: '983GE',
-      authoritativeName: 'San Diego VA medical center 3',
-      city: 'San Diego',
-      stateAbbrev: 'CA',
-      rootStationCode: '983',
-      parentStationCode: '983',
-      requestSupported: true,
-    },
+const typeOfCareId = '323';
+const typeOfCare = 'Primary Care';
+
+const requestFacilityAttributes = getRequestEligibilityCriteriaMock()
+  .attributes;
+
+const facilityIds = ['983', '983GC', '983GB', '983HK', '983QA', '984'];
+
+const requestFacilities = facilityIds.map(id => ({
+  id,
+  attributes: {
+    ...requestFacilityAttributes,
+    id,
+    requestSettings: [
+      {
+        ...requestFacilityAttributes.requestSettings[0],
+        id: typeOfCareId,
+        typeOfCare,
+      },
+    ],
   },
-  {
-    id: '984GF',
-    attributes: {
-      ...getFacilityMock().attributes,
-      institutionCode: '984GF',
-      authoritativeName: 'San Diego VA medical center 4',
-      city: 'San Diego',
-      stateAbbrev: 'CA',
-      rootStationCode: '984',
-      parentStationCode: '984',
-      requestSupported: true,
-    },
+}));
+
+const directFacilityAttributes = getDirectBookingEligibilityCriteriaMock()
+  .attributes;
+
+const directFacilities = facilityIds.map(id => ({
+  id,
+  attributes: {
+    ...directFacilityAttributes,
+    id,
+    coreSettings: [
+      {
+        ...directFacilityAttributes.coreSettings[0],
+        id: typeOfCareId,
+        typeOfCare,
+      },
+    ],
   },
-  {
-    id: '984GG',
-    attributes: {
-      ...getFacilityMock().attributes,
-      institutionCode: '984GG',
-      authoritativeName: 'San Diego VA medical center 5',
-      city: 'San Diego',
-      stateAbbrev: 'CA',
-      rootStationCode: '984',
-      parentStationCode: '984',
-      requestSupported: true,
+}));
+
+const vhaIds = facilityIds.map(
+  id => `vha_${id.replace('983', '442').replace('984', '552')}`,
+);
+
+const facilities = vhaIds.map((id, index) => ({
+  id,
+  attributes: {
+    ...getVAFacilityMock().attributes,
+    uniqueId: id.replace('vha_', ''),
+    name: `Fake facility name ${index + 1}`,
+    address: {
+      physical: {
+        ...getVAFacilityMock().attributes.address.physical,
+        city: `Fake city ${index + 1}`,
+      },
     },
   },
-];
+}));
 
 describe('VAOS integration: VA flat facility page - multiple facilities', () => {
   beforeEach(() => mockFetch());
   afterEach(() => resetFetch());
 
   it('should display list of facilities with show more button', async () => {
     mockParentSites(parentSiteIds, [parentSite983, parentSite984]);
-
-    parentSiteIds.forEach(id => {
-      mockSupportedFacilities({
-        siteId: id,
-        parentId: id,
-        typeOfCareId: '323',
-        data: facilities.filter(f => f.attributes.parentStationCode === id),
-      });
-    });
-
+    mockDirectBookingEligibilityCriteria(parentSiteIds, directFacilities);
```

Comment:
Might be worth in the future seeing if we could wrap these in a higher level helper that takes in a list of facilities and  what types of care they support and builds out all the fetches for you.